### ISSUE TITLE
[Bug]: New releases not including binaries

### ISSUE THREAD

**Issue by mbevc1** [kubernetes-sigs/aws-iam-authenticator#657]
### What happened?

Newer releases, [v0.6.12](https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/tag/v0.6.12) and later don't include binary artefacts.

### What you expected to happen?

Binary artefacts to install :)

### Installation tooling

other (please specify in description)

### AWS IAM Authenticator client version

v0.6.11

### Client information

```Markdown
- OS/arch: Fedora Linux 39
- kubernetes client & version: n/a
```

### Kubernetes API Version

/

**Comment by andybritton-careplanner** [comment]
Looks like this step has been missed https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/master/docs/RELEASE.md#run-goreleaser

**Comment by pieterocp** [comment]
From the looks of it, the  create-release workflow exists but [hasn't been run yet](https://github.com/kubernetes-sigs/aws-iam-authenticator/actions/workflows/create-release.yml)? 
But the release process seems to dictate that this is something to [be done manually](https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/release-0.6/docs/RELEASE.md) by a human as part of the release process. If there are still manual steps to a release process, then it'd be advisable having a per-release checklist as part of a release PR template.

**Comment by ridoo** [comment]
> From the looks of it, the create-release workflow exists but [hasn't been run yet](https://github.com/kubernetes-sigs/aws-iam-authenticator/actions/workflows/create-release.yml)?

In the meantime, the binaries seem to be present now .. but no workflow [had been run still](https://github.com/kubernetes-sigs/aws-iam-authenticator/actions/workflows/create-release.yml). Seems this is down separately from that workflow?!

**Comment by GMartinez-Sisti** [comment]
I still don't see them, vs the previous release that has 10 assets:

<img width="930" alt="image" src="https://github.com/kubernetes-sigs/aws-iam-authenticator/assets/19713226/0ecfcf54-5c3a-4273-8ea2-d8f5a6391e60">

**Comment by ridoo** [comment]
yes you're right .. seems that I was looking for some other binaries (which were not present before as well).

**Comment by jazzywhit** [comment]
I did a workaround to use 0.6.11 that does have the binaries.
The specifics are here: https://github.com/CircleCI-Public/aws-eks-orb/issues/62#issuecomment-1832519083

**Comment by Nuru** [comment]
To recap: Versions 0.6.12, 0.6.13, and 0.5.20 are all missing artifacts that were present in v0.6.11.

It looks like the workflow is intended to have `tag-release` tag a release, push the tag, and have the push trigger the `Create Release` workflow. However, as a safety measure, [workflows do not trigger other workflows when using the default workflow permissions](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow). So the `Create Release` workflow is not getting run.

The good news is that `Create Release` has a `workflow_dispatch` trigger, so at least the latest releases on the 2 branches can be manually built.

**Comment by nckturner** [comment]
Added binaries to latest releases, will look into fixing the release workflow.

**Comment by cpboyd** [comment]
Not sure if the `Create Release` workflow was fixed or not, but the new releases also don't yet have the binaries.

**Comment by psalaberria002** [comment]
Can someone reopen this issue please?
